pex/package.py

```diff
@@ -164,6 +160,20 @@ def __init__(self, url, **kw):
     if self._raw_version is None or self._py_version is None:
       raise self.InvalidPackage('url with .egg extension but bad name: %s' % url)
 
+    abi_tag = 'none'
+    tag_platform = self._platform or 'any'
+    self._supported_tags = set()
+    self._supported_tags.add(('py' + self._py_version.replace('.', ''), abi_tag, tag_platform))
+    # Work around PyPy versions being weird in pep425tags.get_supported
+    if self.py_version == '2.7':
+      self._supported_tags.add(('pp2', abi_tag, tag_platform))
+    elif self.py_version == '3.2':
+      self._supported_tags.add(('pp321', abi_tag, tag_platform))
+    elif self.py_version == '3.3':
+      self._supported_tags.add(('pp352', abi_tag, tag_platform))
```

Comment:
`3.3` vs `352`... intentional? Maybe comment worthy?

Reply:
these are PyPy ABI tags (`pp` -> `PyPy`), so they don't map exactly to CPython/`cp`. `PyPy 3.3` was implicitly `pp352` in PyPy versioning - see: http://doc.pypy.org/en/latest/release-pypy3.3-v5.2-alpha1.html

also commented.